packages/pwa-kit-dev/src/ssr/server/build-dev-server.js

```diff
@@ -139,10 +139,12 @@ export const DevServerMixin = {
 
         app.use('/mobify/bundle/development', app.__devMiddleware)
 
+        app.__hmrMiddleware = (_, res) => res.status(409).send('Hot Module Reloading is disabled.')
```

Comment:
I'm not sure if 409 is the right status.. not familiar with it. Does it fit with what we're seeing?
"request could not be processed because of conflict in the current state of the resource, such as an [edit conflict](https://en.wikipedia.org/wiki/Edit_conflict) between multiple simultaneous updates."

On 2nd thought, I couldn't find any other status that fits better than 409. And "conflict in the current state of the resource" sounds to make more sense now. Yeah, let's keep it.

Reply:
Just FYI - let's keep in mind that there is more than 1 scenario where hmr can be disabled:
- disabled by us in certain templates like the express template: it's because there is no `client` webpack config generated (due to the lack of app/main.js entry point)
- disabled by end users - they toggle off the hmr feature (via a toggle functionality that's coming soon)